### ISSUE TITLE
Add Feature: Physical RFID kanban cards

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#1126]
https://media.ccc.de/v/1426-connected-kanban

**Comment by mayjs** [comment]
Isn't this something that could be added using an external tool accessing the REST-API?

**Comment by xet7** [comment]
@mayjs 

Yes. In that talk there is probably links for code and hardware how to get it working.
